build.zig

```diff
@@ -6,11 +6,50 @@ const Mode = std.builtin.Mode;
 const config = @import("./src/config.zig");
 const Shell = @import("./src/shell.zig");
 
+// TigerBeetle binary requires certain CPU feature and supports a closed set of CPUs. Here, we
+// specify exactly which features the binary needs. Client shared libraries might be more lax with
+// CPU features required.
+const supported_targets: []const CrossTarget = supported_targets: {
+    @setEvalBranchQuota(100_000);
+    var result: []const CrossTarget = &.{};
+    const triples = .{
+        "aarch64-linux",
+        "aarch64-macos",
+        "x86_64-linux",
+        "x86_64-macos",
+        "x86_64-windows",
+    };
+    const cpus = .{
+        "baseline+aes+neon",
+        "baseline+aes+neon",
+        "x86_64_v3+aes",
+        "x86_64_v3+aes",
+        "x86_64_v3+aes",
+    };
```

Comment:
An array of `CrossTarget.ParseOptions` (or just tuples) might be more readable, though it probably doesn't matter with only 5 pairs.

Reply:
I've actually recently started to use "columnar" format for small tables in the source code to _improve_ readability, as it requires less symbols and usually formats better, so this is intentional

Reply:
(Related: https://github.com/ziglang/zig/issues/17145)